### PR TITLE
align dropdown with start as default, useVModel passive to have local…

### PR DESCRIPTION
**Pull request by xvaara**
I think it's better to start from start and to not require v-model to be set.

### CODE REVIEW

**Comment by VividLemon** [comment]
So vue use vmodel "passive" option essentially means that you don't actually need to bind a model value?

**Comment by VividLemon** [comment]
Interesting behavior when using passive. I extended the same to BCarousel. We should figure out the best way to communicate in the docs that in some components, v-model is optional

File: packages/bootstrap-vue-next/src/components/BDropdown/BDropdown.vue [hunk 1]
```diff
@@ -169,7 +171,7 @@ const floatingPlacement = computed(() =>
     bottom: !dropupBoolean.value,
     start: dropstartBoolean.value,
     end: dropendBoolean.value,
-    dropstart: alignStartBoolean.value,
+    dropstart: !alignCenterBoolean.value,
     dropend: alignEndBoolean.value,
```
Comment (VividLemon) [inline packages/bootstrap-vue-next/src/components/BDropdown/BDropdown.vue:175]: What happens if you do _alignCenter_ and _alignEnd_?

**Comment by xvaara** [comment]
> So vue use vmodel "passive" option essentially means that you don't actually need to bind a model value?

It basically creates a ref in the background and has a watch on it with a emit in it.

File: packages/bootstrap-vue-next/src/components/BDropdown/BDropdown.vue (hunk 1 above)
Comment (xvaara) [reply packages/bootstrap-vue-next/src/components/BDropdown/BDropdown.vue:175]: I looked in to this and opened a can of worms... The alignments can be breakpoint dependant. Do we want to support that? I'll take a look at the bootstrap docs with a thought.

Comment (xvaara) [reply packages/bootstrap-vue-next/src/components/BDropdown/BDropdown.vue]: I redid this. Now we can use end and center on dropstart and dropend also. So we have all the floating-ui directions.

**Pull request merged**